devices/src/virtio/block.rs

```diff
@@ -654,11 +654,10 @@ impl VirtioDevice for Block {
 
 #[cfg(test)]
 mod tests {
-    extern crate tempfile;
+    extern crate utils;
+    use utils::tempfile::TempFile;
 
-    use self::tempfile::{tempfile, NamedTempFile};
     use super::*;
-
     use libc;
     use memory_model::Address;
     use std::fs::{metadata, OpenOptions};
```

Comment:
I think it would be nice to have uniformity also in tests and keep separate the `extern`s, the `use`s from `std` and the ones from other crates.

Reply:
Updated. Take a look: [devices/src/virtio/block.rs:657](https://github.com/firecracker-microvm/firecracker/blob/f1754fb3c54889c98231bfe3cdcf023c3d74fcf3/devices/src/virtio/block.rs#L657)

Reply:
:thumbsup: Looks great now.